core/src/main/java/io/questdb/griffin/FunctionFactoryDescriptor.java

```diff
@@ -208,6 +211,25 @@ public static short toType(int mask) {
         return (short) (mask & TYPE_MASK);
     }
 
+    public static String translateSignature(String signature) throws SqlException {
+        int openBraceIndex = validateSignatureAndGetNameSeparator(signature);
+        StringSink sb = Misc.getThreadLocalBuilder();
+        sb.put(signature);
+        sb.clear(openBraceIndex + 1);
+        for (int i = openBraceIndex + 1, n = signature.length() - 1; i < n; i++) {
+            char c = signature.charAt(i);
+            String type = typeNameMap.get(c | 32);
+            if (type == null) {
+                throw new IllegalArgumentException("offending: '" + c + '\'');
+            }
```

Comment:
Isn't adding "var" a bit too verbose ? 
`int` vs `const int` should be well readable I think . 

btw,
```
).put(type) 
```
would be a bit better than  `+ type ` .

Reply:
addressed in a1a73ca10077447f80b5c96f5ce3d8553ceb85f5